src/component_types/character.lua

```diff
@@ -57,6 +57,9 @@ return {
         this.direction     = 'right'
         this.nade          = nil
         this.throw_enabled = false
+        this.squish        = 0
+        this.squishiness   = this.squishiness or 1
+        this.squishspeed   = 32 -- pixels per second
```

Comment:
Changed this line to 32 to slow down squish speed. At 64 it was a little too fast, now it's obvious it's a squish effect.